drivers/misc/intel/comm_widget/intel_comm_widget_messages.c

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <errno.h>
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "intel_comm_widget.h"
+#include "pmc_interface.h"
+
+/*
+ * Report number of used HP-SRAM memory banks to the PMC, unit is 32 KB.
```

Comment:
I'd replace 'Report' with 'Tell'.  Report made me think this is returning that value.